services/121-service/src/events/dto/get-event.dto.ts

```diff
@@ -3,7 +3,7 @@ import { WrapperType } from '@121-service/src/wrapper.type';
 import { ApiProperty } from '@nestjs/swagger';
 
 class AttributesDto {
-  [key: string]: string;
+  [key: string]: string | unknown;
```

Comment:
I feel like this change might come back to haunt you, and you might have to adjust it further, but it is ok to merge it as is for now.

Reply:
What do you think about this:
class AttributesDto {
  [key: string]: string | number | boolean | unknown;
}

Reply:
We have to talk about this change, because in we have different types and I think this will be good to be:
[key: string]: string | number | unknown;

Reply:
I think, as far as typescript is concerned, doing `string | unknown` or `string | number | unknown` is the same as just doing `unknown` because it still doesn't know what type it is.

When I say 

> I feel like this change might come back to haunt you

it is because I think this is used in a lot of places, and if you change it to `unknown` then you have to do type checking in a lot of places.

I would leave it as `string | number` if possible. But you can decide what is best (you have more context) and then change it accordingly.

Reply:
Then, maybe is better to use null, because we have this:
![image](https://github.com/user-attachments/assets/adec002d-6559-4dcd-92af-785cf0f87ee1)

Reply:
Ah yes, then `string | null` should be good indeed

Reply:
Done.